src/components/BrideReview/BrideReview.js

```diff
@@ -0,0 +1,293 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import styled from "styled-components";
+import { ThemeProvider } from "styled-components";
+import Button from "../Button/Button";
+import PinkButton from "../Button/Button";
+
+const BridalReviews = styled.div`
+  width: 100%;
+  background-color: ${props => props.theme.colors.backgroundGray};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    height: 300px;
+  }
+`;
+
+const Container = styled.div`
+  display: flex;
+  height: 240px;
+  justify-content: space-around;
+  background-color: ${props => props.theme.colors.backgroundGray};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    height: 327px;
+  }
+`;
+
+const ReviewContainer1 = styled.div`
```

Comment:
Instead of have `ReviewContainer1`, `ReviewContainer2` and `ReviewContainer3` - can you just have one `ReviewContainer`? They all seem to share the same properties. Same goes for your other styled components here - I feel like you can make this DRYer.

Reply:
That makes sense.  I think I need to have one for mobile and desktop so the desktop ones stay `display: none;`d until it is a big enough screen.  There may be a better, dryer sol'n that I am not thinking of...Changed ReviewContainer[n] to ReviewContainerMobile and ReviewContainerDesktop and the same with other elements.